sdk/schemaregistry/schema-registry/src/conversions.ts

```diff
@@ -58,7 +59,4 @@ function convertResponse<T>(response: GeneratedResponse, additionalProperties: T
     serializationType: response.serializationType!,
     ...additionalProperties
   };
-
-  Object.defineProperty(converted, "_response", { value: response._response, enumerable: false });
```

Comment:
This is more for my education about core v2. Can the caller still get to raw response details like headers in some way?

Reply:
Yes!, `OperationOptions` now has an `onResponse` property that the customer can pass a callback to which takes the raw response in the first parameter and the flatten response in the second. You can see it in action in the updated tests here: https://github.com/Azure/azure-sdk-for-js/blob/c215ee38b12533475649685253c4477f0ca7da28/sdk/schemaregistry/schema-registry/test/schemaRegistry.spec.ts#L15-L19

Reply:
Nice.